test/lit/passes/outlining.wast

```diff
@@ -0,0 +1,95 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+
+;; RUN: foreach %s %t wasm-opt --outlining -S -o - | filecheck %s
+
+(module
+	;; CHECK:      (type $0 (func (result i32)))
+
+	;; CHECK:      (type $1 (func (param i32)))
+
+	;; CHECK:      (func $a (result i32)
+	;; CHECK-NEXT:  (call $outline$5
+	;; CHECK-NEXT:   (i32.const 7)
+	;; CHECK-NEXT:  )
+	;; CHECK-NEXT:  (return
+	;; CHECK-NEXT:   (i32.const 4)
+	;; CHECK-NEXT:  )
+	;; CHECK-NEXT: )
+	(func $a (result i32)
```

Comment:
Normally we just have just one or two spaces of indent. We try to be consistent within files, but we're definitely not consistent between files.

Reply:
done